initializers/db.go

```diff
@@ -0,0 +1,42 @@
+package initializers
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"tiny-site-backend/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+var DB *gorm.DB
+
+func ConnectDB(config *Config) {
+	var err error
+	dsn := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=disable TimeZone=Asia/Shanghai", config.DBHost, config.DBUserName, config.DBUserPassword, config.DBName, config.DBPort)
+
+	DB, err = gorm.Open(postgres.Open(dsn), &gorm.Config{})
+	if err != nil {
+		log.Fatal("Failed to connect to the Database! \n", err.Error())
+		os.Exit(1)
+	}
+
+	if err := DB.Exec("CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\"").Error; err != nil {
+		log.Fatal("Failed to create uuid-ossp extension:  \n", err.Error())
+		os.Exit(1)
+	}
+
+	DB.Logger = logger.Default.LogMode(logger.Info)
+
+	log.Println("Running Migrations")
```

Comment:
Hey @vinit717 for the logs, where do we plan to store them? In a file or a database for future auditing purposes..

Reply:
Don't know about this where we can store the logs and how 
what is your opinion here?

Reply:
If we want for an audit purpose, this depends on what logs are we printing then a database helps. If we are doing in files then **the files have to be managed** and mostly the data has to be pushed to some long term storage.

We can ask @iamitprakash I feel

Reply:
Ok